config/addons.yml

```diff
@@ -0,0 +1,8 @@
+# example usage
+#dplan_addons:
+#    -
+#      name: "mygithuborga/myaddon"
+#      version: "0.6.1"
```

Comment:
here I would like to add the v in front: ```version: "v0.6.1"```

Reply:
indeed, thx [e82c204](https://github.com/demos-europe/demosplan-core/pull/3633/commits/e82c204555a821fc8382c1a3eef6ce45e8b96860)